test/skate_web/controllers/page_controller_test.exs

```diff
@@ -52,6 +52,29 @@ defmodule SkateWeb.PageControllerTest do
       assert html =~ "selected_route_ids&quot;:[&quot;1&quot;]"
     end
 
+    @tag :authenticated
+    test "includes user test groups in HTML", %{conn: conn, user: user} do
+      user_struct = user |> Skate.Settings.User.get()
+
+      Skate.Settings.TestGroup.update(%{
+        Skate.Settings.TestGroup.create("html-test-group")
+        | users: [user_struct]
+      })
+
+      user_struct = user |> Skate.Settings.User.get() |> Skate.Repo.preload(:test_groups)
+
+      conn = get(conn, "/")
+
+      html = html_response(conn, 200)
+
+      json =
+        Jason.encode!(user_struct.test_groups |> Enum.map(& &1.name))
```

Comment:
I'm also kinda concerned the test needs to handle the case of having multiple groups or groups returned in different orders?

If so maybe it needs to be a different data type?